src/style.css

```
@@ -0,0 +1,109 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  display: flex;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  height: 70px;
+  margin: 20px 0;
+  width: 100%;
+}
+
+.icon-form {
+  display: flex;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  height: 42px;
+  margin: 20px 0;
+  width: 100%;
+}
```

```suggestion
.form {
  display: flex;
  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
  height: 70px;
  margin: 20px 0;
  width: 100%;
}

.form--small {
  height: 42px;
}
```
this is how it should works
you have common class `form` and modifier `form--small` for small form, where you specify only differences